ws/index.js

```diff
@@ -9,4 +9,16 @@ fastify.register(require('fastify-websocket'), {
   handle: conn => conn.pipe(conn),
 })
 
+fastify.get('/', {
+  websocket: true
+}, (connection /* SocketStream */ , req /* FastifyRequest */ ) => {
+  connection.socket.on('message', message => {
+    fastify.websocketServer.clients.forEach((client) => {
+      if (client.readyState === 1) {
+        client.send(message)
+      }
+    })
```

Comment:
question: are we sending this to the sender too?

Reply:
for the purpose of this i was sending it to the sender as well. but i can try to filter the clients in some manner.

Reply:
aren't we drawing twice if we're not excluding the sender from this loop?